sdk/servicebus/service-bus/test/internal/serviceBusClient.spec.ts

```diff
@@ -147,4 +149,41 @@ describe("serviceBusClient unit tests", () => {
       }
     });
   });
+
+  describe("createConnectionContext", () => {
```

Comment:
These are good tests. I think that if you've got time it'd be good to add tests one level up (from the `createConnectionContextForTokenCredential` and `createConnectionContextForConnectionString` level) since that's where the bug originated from.

Reply:
Fine. Added.